components/Datepicker/react.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import Calendar from 'react-modern-calendar-datepicker'
+import 'react-modern-calendar-datepicker/lib/DatePicker.css'
+import classNames from 'classnames/bind'
+import styles from './style.module.scss'
+
+const cx = classNames.bind(styles)
+
+const Datepicker = ({
+	value,
+	onChange,
+	inputPlaceholder,
+	disabled,
+	calendarPopperPosition,
+	calendarClassName,
+	...props
+}) => {
+	return (
+		<div className={cx('datepicker')}>
```

Comment:
It might be good to include a className prop to the MDL component that then gets passed into this div to let the user layer in more styling if necessary.